apps/sveltekit-example-app/src/hooks.server.ts

```diff
@@ -36,7 +36,8 @@ type ValidatedSession =
       user: User
     }
 
-const authHandler: Handle = async ({ event, resolve }) => {
+const authHandler: Handle = async (input) => {
```

Comment:
Upgrading to eslint-typescript 8.9 triggered the @typescript-eslint/unbound-method rule on this line. It was referring to the `resolve` in the destructured input.  The rule seems legit but unsure why it triggered after the upgrade and not before. Worked around it by not destructuring the input in the method signature.